packages/nocodb-sdk/src/lib/sqlUi/MssqlUi.ts

```diff
@@ -138,9 +138,9 @@ export class MssqlUi {
     ];
   }
 
-  static getNewColumn(suffix) {
+  static getNewColumn(suffix: number, uidt: UITypes) {
     return {
-      column_name: 'title' + suffix,
+      column_name: `Untitled${uidt}${suffix !== 0 ? suffix : ''}`,
```

Comment:
instead of checking suffix here. you can return empty string in `generateUniqueColumnSuffix` to keep the code more clean. (apply to all)